newapi/super/S_API/bot.py

```
@@ -32,7 +32,9 @@ def Add_To_Bottom(self, text, summary, title, poss="Head|Bottom"):
         printe.test_print(f"** Add_To_Bottom .. [[{title}]] ")
         # printe.showDiff("", text)
         # ---
-        ask = self.ask_put(newtext=text, message=f"** Add_To {poss} .. [[{title}]] ", job="Add_To_Bottom", username=self.username, summary=summary)
+        user = self.username or self.user_login
```

Pylint (3.3.7)

[error] Instance of 'BOTS_APIS' has no 'user_login' member

(E1101)